### DISCUSSION TITLE
Connection breaks after some time

### DISCUSSION THREAD

**Discussion by wireva** [miguelgrinberg/python-socketio#743] [Q&A]
![image](https://user-images.githubusercontent.com/46248939/120894973-4345da00-c61b-11eb-93a7-892cf168bb83.png)

I have a game and after a minute or so the game hangs (because the sid changed). The connection breaks after some misformatted messages. Why is that? I use nuxt-socketio and python-socketio.

**Comment by miguelgrinberg** [comment]
Have you looked at the logs for clues? See the troubleshooting section of the documentation to learn how to enable logs if you don't know how to.
